core/sqf/export/include/dtm/tm.h

```diff
@@ -274,11 +274,19 @@ typedef struct TM_HBASEREGIONINFO {
 // Trafodion start //
 #define TM_MAX_REGIONSERVER_STRING 2048
 #define TM_MAX_DDLREQUEST_STRING 2048
```

Comment:
Current usage is 1k. There is some more room for additional options. Will make this configurable as part of separate checkin.